### DISCUSSION TITLE
Avalanche of connection error notifications

### DISCUSSION THREAD

**Discussion by Bbillyben** [inventree/InvenTree#5235] [General]
Hello All, 

I'm hosting a test semi-prod instance of inventree (just upgraded to latest btw) on a machine in my organization.

They have a bot that scans VMs to check for a ton of known security flaws. 

However, this produces a staggering number of connection error warnings in Inventree (around 2k every bot pass).

The issue is that the application slows to a crawl when there are so many notifications, leading to the workers failing repeatedly.

I must then restart the containers (or the vm....) and access the site admin to remove the notifications,

Of course, it's a low conf VM  - 2 cpu, 2gb ram - which host another django site.

Is there any workaround I can use ? 

note : I've upgraded to check wether it solve that issue.
[edit] I've lowered the nbr of days for Notification Deletion Interval, i will see if it has any good effect.

# Version Information:
InvenTree-Version: 0.13.0 dev
Django Version: 3.2.20
Commit Hash: 01f2aa5
Commit Date: 12/07/2023

Database: postgresql
Debug-Mode: False
Deployed using Docker: True
Platform: Linux-5.10.0-22-amd64-x86_64-with
Installer: DOC

Active plugins: [{'name': 'InvenTreeBarcode', 'slug': 'inventreebarcode', 'version': '2.0.0'}, {'name': 'InvenTreeCoreNotificationsPlugin', 'slug': 'inventreecorenotificationsplugin', 'version': '1.0.0'}, {'name': 'StockTestPanel', 'slug': 'stocktest', 'version': '0.1'}, {'name': 'StockTrackPanel', 'slug': 'stocktrack', 'version': '0.1'}]

**Comment by matmair** [comment]
Depending on what solution your IT uses this might be an active one that tries to DDOS InvenTree. There is not much we can do about that - if you are DDOSes without a loadbalancer in front the app will experience connection timeouts eventually. You an but a loadbalancer with throtteling infront or ask IT/sec to not kill your production setup.
